spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -275,15 +279,17 @@
 
   let(:expected_presentation_location) {
```

Comment:
Let's rename that to `:expected_extras`.

Reply:
Yes, I missed this one. Thank you for catching!